handlers/overview.go

```diff
@@ -34,6 +34,7 @@ func filterFlexOverview(w http.ResponseWriter, req *http.Request, rc RenderClien
 	var filterJob *filter.GetFilterResponse
 	var fErr, dErr, fdsErr, imErr error
 	var isMultivariate bool
+	var supVar string
```

Comment:
What is the purpose of supVar?
It doesn't seem to get assigned (line 246) until after it has been used (line 210)

Reply:
Line `210` is within a function that's called on line `250` (after the assignment on line `246`)